apps/daimo-mobile/src/view/screen/send/SendTransferButton.tsx

```diff
@@ -40,14 +43,16 @@ export function SendTransferButton({
   memo,
   minTransferAmount = 0,
   route,
+  landlineBankTransferOption,
 }: {
   account: Account;
-  recipient: EAccountContact | BridgeBankAccountContact;
+  recipient: EAccountContact | LandlineBankAccountContact;
   dollars: number;
   toCoin: ForeignToken;
   memo?: string;
   minTransferAmount?: number;
   route?: ProposedSwap | null;
+  landlineBankTransferOption?: BankTransferOptions | null;
```

Comment:
it looks like this prop is for deposit vs withdraw

so far, we've used `SendTransferButton` for sending onchain transfers. for landline, this would fit nicely with Landline withdrawal = sending USDC to a landline receiving addres.

may be worth keeping the button on the deposit-from-landline screen separate.